.circleci/config.yml

```diff
@@ -35,6 +35,7 @@ workflows:
             branches:
               only:
                 - main
+                - release-v2.*
```

Comment:
```suggestion
                - release-v.*
```
I think we can keep it more generic for all release branches. We still support v1 branches after all.

Reply:
make sense 👍